### PR TITLE
Feat/drunemeton 1

### PR DESCRIPTION
**Pull request by amimart**
Forge the base genesis for the new `okp4-drunemeton-1` network.

## 💎 Genesis details

- **Chain ID**: `okp4-drunemeton-1`
- **Launch time**: `2024-02-06T15:00:00Z`

Please find below the notable details on the different modules parameters and state of the genesis. If not mentioned, consider the default parameters.

### Initial supply

The initial supply respects the planned one for mainnet: `200000000000000uknow`.

It has been assigned entirely to a core team account.

### Consensus

The only modified consensus parameters is the maximum gas allowed per bock: `20000000`.

### Staking

Only `31` validators allowed in the active set.

### Slashing

Same parameters as `okp4-nemeton-1` that seems to fit well, default ones being too strict.

### Governance

Set the voting period to 24h for a standard proposal, 12h for and expedited proposal.

### Logic

Consider max gas to `20000000`.

### Wasm

Permissioned settings: only the core team address can store contract codes, or the gov module.

## 📥 Register process

The issue template for registering a genesis validator has been updated to manage this new network, the registering is open until `2024-02-04 00:00:00Z`, but we'll certainly close before.

## ✋ `okp4-nemeton-1`

By adding the new `okp4-drunemeton-1` network, the previous `okp4-nemeton-1` becomes deprecated and marked as discontinued.

## 🤖 CI Automation

CI scripts have been refactored with the new CLI okp4d `v6.0.0` commands.

### CODE REVIEW

File: chains/drunemeton-1/genesis.json [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+          "bootstrap": "",
+          "virtual_files_filter": {
+            "whitelist": [],
+            "blacklist": []
+          }
+        },
+        "limits": {
+          "max_gas": "2000000",
+          "max_size": "5000",
+          "max_result_count": "1",
```
Comment (ccamel) [inline chains/drunemeton-1/genesis.json:219]: I'd like to increase the possible number of results to 2 (or 3). Some test scenarios may require this.

File: chains/drunemeton-1/genesis.json [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+          "allow_messages": ["*"]
+        }
+      }
+    },
+    "logic": {
+      "params": {
+        "interpreter": {
+          "predicates_filter": {
+            "whitelist": [],
+            "blacklist": []
```
Comment (ccamel) [inline chains/drunemeton-1/genesis.json:208]: No predicate blacklist here?

**Review by ccamel — CHANGES_REQUESTED** [review]
That's great. :smiley: 
Just minor remarks.

File: chains/drunemeton-1/genesis.json (hunk 2 above)
Comment (amimart) [reply chains/drunemeton-1/genesis.json:208]: You're right, I'll change that!

Comment (amimart) [reply chains/drunemeton-1/genesis.json:308]: I've set the same whitelist the current one on `okp4-nemeton-1`

**Review by ccamel — APPROVED** [review]
:rocket:

**Pull request merged**